docs/proposals/apb-state-support.md

```diff
@@ -0,0 +1,60 @@
+# Abstract
+
+This proposal will outline a simple and secure method for allowing APB developers to set state during a APB
```

Comment:
There is an ongoing discussion about the APB as an instance of a more generic Service Bundle (Bundle). This is the reason for @shawn-hurley earlier comment about "bundle contract".

My gut says that we should say Service Bundle or Bundle when referring to the generic and only use APB when we are specifically talking about Ansible Playbook Bundles.